packages/wonder-blocks-dropdown/components/dropdown.md

```diff
@@ -0,0 +1,233 @@
+
+### Custom opener attached to Dropdown
+
+A Dropdown allows various elements to be used as dropdown menus. In this example we use the IconButton to open a Dropown of items to select from.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
+const {icons} = require("@khanacademy/wonder-blocks-icon");
+const {StyleSheet} = require("aphrodite");
+
+const dropdownItems = [
+    <ActionItem
+        label="Profile"
+        href="http://khanacademy.org/profile"
+        testId="profile"
+    />,
+    <ActionItem
+        label="Teacher dashboard"
+        href="http://khanacademy.org/coach/dashboard"
+        testId="dashboard"
+    />,
+    <ActionItem
+        label="Settings"
+        onClick={() => console.log("user clicked on settings")}
+        testId="settings"
+    />,
+    <ActionItem
+        label="Help"
+        disabled={true}
+        onClick={() => console.log("this item is disabled...")}
+        testId="help"
+    />,
+    <ActionItem
+        label="Feedback"
+        disabled={true}
+        href="/feedback"
+        testId="feedback"
+    />,
+    <SeparatorItem />,
+    <ActionItem
+        label="Log out"
+        href="http://khanacademy.org/logout"
+        testId="logout"
+    />,
+    <OptionItem
+        label="Show homework assignments" value="homework"
+        onClick={() => console.log(`Show homework assignments toggled`)}
+    />,
+    <OptionItem
+        label="Show in-class assignments" value="in-class"
+        onClick={() => console.log(`Show in-class assignments toggled`)}
+    />
+];
+
+<Dropdown
+    disabled={false}
+    menuText="Betsy Appleseed"
+    testId="teacher-menu"
+    menuItems={dropdownItems}
+>
+    {(openDropdown) => (
+        <IconButton
+            icon={icons.caretDown}
+            aria-label="search"
+        />
+    )}
+</Dropdown>
+```
+### Typography based openers
+
+In this example we use a Title component with a Dropdown. You can define custom styles that will take place on different mouse events that ClickableBehavior supports.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
+const {icons} = require("@khanacademy/wonder-blocks-icon");
+const {StyleSheet} = require("aphrodite");
+const {Title} = require("@khanacademy/wonder-blocks-typography");
+
+// Custom styles for mouse events can be defined like so
+const styles = StyleSheet.create({
+    focused: {
+        border: "none",
+    },
+    hovered: {
+        textDecoration: "underline",
+    },
+    pressed: {},
```

Comment:
Can you modify the `focused` and `pressed` states to be more visually different from the default and `hovered` states?  Maybe the `pressed` state changes the color of the the text and maybe focused gives it a background color.